rules/execute-command.js

```diff
@@ -59,15 +59,17 @@ async function run() {
       acc.push(next);
       return acc;
     }, []);
-
+    
+    dists.sort();
+    
     const dirsString = dists.map(item => `"${item}"`).join(' ');
 
     for (const dist of dists) {
       if (!exists(join(main, dist))) {
         exec(`mkdir -p "${dist}"`, {cwd: main});
       }
     }
-    exec(`tar czf "${out}" ${dirsString}`, {cwd: main, stdio: 'inherit'});
+    exec(`tar czf "${out}" --sort=name ${dirsString}`, {cwd: main, stdio: 'inherit'});
```

Comment:
is there some way to add a test for this?

Reply:
Added in PR. To test that the test is correctly testing the right behavior uncomment the `dists.sort();` and then run `yarn test testTarSortOrderListsParentDirectoriesFirst` and it will error.

Reply:
Just some two cents, would it be easier to add `{recursive: true}` in the mkdir call at https://github.com/uber-web/jazelle/blob/main/rules/untar.js#L73 and then test `copy` via child_process.spawn or .fork instead of doing a full-on end-to-end test w/ bazel and yarn lockfile and all that stuff in the picture?